atd-vzd/schema/engineering_areas.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE public.engineering_areas(
+    area_id integer,
```

Comment:
idk if this is repetitive again  and if it should just be `id` but i feel like `area_id` helps make it more clear that this isnt a random id we are creating but that we pull it from the data itself? idek

Reply:
since there's a decent chance this data will be refreshed from the same source down the road, i would lean toward preserving the column names as they are in the source data, but i draw the line at capitalization. so this column would be `engineering_area_id`.

Reply:
derp, i see Frank already made a comment with a different approach. sorry for the back n forth. ignore me...